typescript/src/schema/proto.ts

```diff
@@ -586,12 +586,16 @@ function augmentMethod(
 
 export function getSingleHeaderParam(
   rule: protos.google.api.IHttpRule
-): string[] {
+): string[][] {
```

Comment:
Is there any reason why it must be a 2d array and not just a string array of all matches?

Reply:
This method is used to support multiple variable in format like `'{param1.param2=abc/*/foo/*/}`. This method separate by "." and return array like `[param1, param2]`.

After 5 minutes thought, I think we could flat the 2 d array. Let me try out.

Reply:
I see now, thank you!